admin/src/Game/List.tsx

```diff
@@ -0,0 +1,20 @@
+import {
+  List,
+  Datagrid,
+  TextField,
+  NumberField,
+  BooleanField,
+} from "react-admin";
+import { GameUser } from "./User";
+
+export const GameList = (props: any) => (
+  <List {...props}>
+    <Datagrid rowClick="show">
+      <TextField source="id" />
+      <NumberField source="currentPlayer" label="Curent id player" />
+      <GameUser />
```

Comment:
Why GameUser has no source props ? it is better to have at the same level all the source props